### DISCUSSION TITLE
🐛 BUG: options tooltip disappears if you scroll

### DISCUSSION THREAD

**Discussion by rafalnawojczyk** [dailydotdev/daily#1278] [General]
### What went wrong? 🤔

Tooltip disappears when you use your scroll wheel

### Expected Behavior

Persist while scrolling

### Steps to Reproduce Issue

```bash
1. Daily dev homepage with articles
2. Hover on article and click three vertical dots in top-right corner
3. Options tooltip appears. Now use your scroll wheel
4. Tooltip disappears
```

### Browsers

Chrome

### OS

Mac

### Version of daily.dev

3.31.17

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @rafalnawojczyk,

Thanks for reporting this issue. Does it affect your experience somehow?

**Comment by rebelchris** [comment]
For now it's intended behaviour, converting to discussion to see other input.
